src/client/App.css

```diff
@@ -39,4 +39,53 @@
 
 ul {
   list-style: none;
+} */
+
+.search-bar {
+  margin-bottom: 40px;
+}
+
+.search-form {
+  margin: 20px 0;
+}
+
+.myButton {
```

Comment:
It's best to pick one coding style - either use camelCase, or kebab-case, but try not to mix and match.

Reply:
Agreed on camelCase- will change all existing code